external/slowparse/slowparse.js

```diff
@@ -1629,15 +1629,15 @@
     // passed the beginning `<tagname` of an HTML element.
     _parseCDATA: function(tagname) {
       var token,
-           matchString = '</'+tagname+'>',
+           matchString = `</${tagname}>`,
            text,
            textInterval = { start: 0, end: 0 },
            openTagEnd = this.domBuilder.currentNode.parseInfo.openTag.end,
            closeTagInterval;
 
       this.stream.makeToken();
       while (!this.stream.end()) {
-        if (this.stream.match(matchString, true)) {
+        if (this.stream.match(matchString, true, true)) {
```

Comment:
What case does this cover? Is there a test for it?

Reply:
I'm not familiar enough with Slowparse to know _exactly_ what it all does, but the comments above the `Stream.prototype.match()` method say
> caseFold can be set to `true` to make the match case-insensitive.

That's what this does here, and immediately fixed the problem when I passed `true` to it. I added tests that try different cases [here](https://github.com/Khan/live-editor/pull/660/files#diff-838964eb6b468cbc568d777be1895122R96), they passed.

Reply:
Ah, so there was previously a change that involved calling .toLowerCase(), and I thought that was how you fixed it. Did you realize you only needed to change line 1640 to fix it, and then reverted the .toLowerCase() calls?

Reply:
Haha, exactly. I had initially tried using .toLowerCase() to no avail, got it working by changing line 1640, then tested some more this morning and found the lowercasing wasn't even needed.

Sorry for any confusion, this PR should have been a lot cleaner... at least I'll learn from it!